src/main/java/com/picolite/controllers/ArticleController.java

```diff
@@ -0,0 +1,49 @@
+package com.picolite.controllers;
+
+
+import com.picolite.fakedb.FakeDB;
+import com.picolite.models.Article;
+import com.picolite.models.ArticleContainer;
+import com.picolite.tools.FancyText;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.servlet.ModelAndView;
+
+@Controller
+public class ArticleController {
+
+
+    @CrossOrigin
+    @RequestMapping(value = "/articles/{id}", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
You add @RequestMapping(value="/articles/") right below the controller annotation and that will make it so you don't have to prefix to all of your RequestMappings with "/articles/"